pkg/vms/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8">
+  <title>Virtual Machines Management</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="../base1/cockpit.css">
+  <link rel="stylesheet" href="app.css">
+  <script type="text/javascript" src="../base1/bundle.js"></script>
+  <script type="text/javascript" src="../base1/jquery.js"></script>
+  <script type="text/javascript" src="../base1/cockpit.js"></script>
```

Comment:
There's no need to load both jquery.js and cockpit.js if you're going to be loading base1/bundle.js ...